accepted/0000-make-scripts-install-opt-in.md

```diff
@@ -0,0 +1,49 @@
+# Transition to a model where install scripts must be explicitly allowed during the install process.
+
+## Summary
+
+Instead of by default always running the install scripts (`preinstall`, `install`, `postinstall`, `prepublish`, `preprepare`, `prepare`, `postprepare`) if they are present during the install process, provide flags to require users to explicitly allow them to run, either whoelsale as "one big swtich", or on a package by package (and optionally version by version) basis. Also provide matching `npm config` options to do the same globally and permanently instead of on every install.
```

Comment:
Please clarify if scripts are *ignored* or *fail* the install if they are not allowed to run.